core/src/core-api/featuretoggles.js

```diff
@@ -54,7 +46,37 @@ class LuigiFeatureToggles {
    * @example Luigi.featureToggles().getActiveFeatureToggleList();
    */
   getActiveFeatureToggleList() {
+    return [...get(this.featureToggleList)].filter(ft => !ft.startsWith('!'));
+  }
+
+  /**
+   * Get a list of active feature toggles
+   * @memberof FeatureToggles
+   * @return {Array} of feature toggles
+   * @example Luigi.featureToggles().getFeatureToggleList();
+   */
+  getFeatureToggleList() {
     return [...get(this.featureToggleList)];
   }
+
+  isValid(featureToggleName) {
+    if (isString(featureToggleName)) return true;
+
+    console.warn('Feature toggle name is not valid or not a type of string');
+    return false;
+  }
+
+  isDuplicateOrDisabled(featureToggleName) {
+    if (get(this.featureToggleList).includes(`${featureToggleName}`)) {
+      console.warn('Feature toggle name already exists');
+      return true;
+    }
+
+    if (get(this.featureToggleList).includes(`!${featureToggleName}`)) {
+      console.warn('Disabled feature toggle can not be activateable');
```

Comment:
```suggestion
      console.warn('Disabled feature toggle can not be activated');
```
'activated' might fit better in this context.
 
(I was wondering if this is even needed. Do we even need to show the user this warning that they cannot activate a feature toggle. I would assume if I am not allowed to access something, it's because I shouldn't. Therefore I would not be necessarily entitled to so much information. What are your thoughts on this ? )